src/components/Table/styles.tsx

```diff
@@ -53,9 +54,12 @@ export interface GridStyleDef {
 
 function memoizedTableStyles() {
   const cache: Record<string, GridStyle> = {};
-  return (props?: GridStyleDef) => {
-    const { inlineEditing = false, grouped = false, rowHeight = "flexible", cellHighlight } = props || {};
-    const key = `${inlineEditing}|${grouped}|${rowHeight}`;
+  return (props: GridStyleDef = {}) => {
+    const { inlineEditing = false, grouped = false, rowHeight = "flexible", cellHighlight = false } = props;
```

Comment:
Nit: could drop this line now? Good call on the sort!

Reply:
I don't think we can drop this line, we still use the destructured props to apply the styles.

Reply:
Oh right :-) Nm!